content/docs/guides/basic-auth.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Basic auth
+---
+
+# Securing Prometheus API and UI endpoints using basic auth
+
+Prometheus does not directly support [basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) (aka "basic auth") for connections to Prometheus instances (i.e. to the expression browser or [HTTP API](../../prometheus/latest/querying/api)). If you would like to enforce basic auth for those connections, we recommend using Prometheus in conjunction with a [reverse proxy](https://www.nginx.com/resources/glossary/reverse-proxy-server/) and applying authentication at the proxy layer. You can use any reverse proxy you like with Prometheus, but in this guide we'll provide an [nginx example](#nginx-example).
+
+NOTE: Although basic auth connections *to* Prometheus instances are not supported, TLS is supported for connections *from* Prometheus instances to [scrape targets](../prometheus/latest/configuration/configuration/#<scrape_config>).
+
+## nginx example
+
+Let's say that you want to run a Prometheus instance behind an [nginx](https://www.nginx.com/) server available at the `example.com` domain (which you own), and for all Prometheus endpoints to be available via the `/prometheus` endpoint. The full URL for Prometheus' `/metrics` endpoint would thus be:
+
+```
+https://example.com/prometheus/metrics
+```
+
+Let's also say that you want to require a username and password from all users accessing the Prometheus [expression browser](/docs/visualization/browser). For this example, use `admin` as the username and choose any password you'd like.
```

Comment:
This configuration controls access to the entire Prometheus, not just the expression browser.